shell/common/gin_converters/content_converter.cc

```diff
@@ -174,11 +174,41 @@ v8::Local<v8::Value> Converter<content::PermissionType>::ToV8(
     v8::Isolate* isolate,
     const content::PermissionType& val) {
   using PermissionType = electron::WebContentsPermissionHelper::PermissionType;
+  // Based on mappings from content/browser/devtools/protocol/browser_handler.cc
+  // Not all permissions are currently used by Electron but this will future
+  // proof these conversions.
   switch (val) {
+    case content::PermissionType::ACCESSIBILITY_EVENTS:
+      return StringToV8(isolate, "accessibility-events");
+    case content::PermissionType::AR:
+      return StringToV8(isolate, "ar");
+    case content::PermissionType::BACKGROUND_FETCH:
+      return StringToV8(isolate, "background-fetch");
+    case content::PermissionType::BACKGROUND_SYNC:
+      return StringToV8(isolate, "background-sync");
+    case content::PermissionType::CLIPBOARD_READ_WRITE:
+      return StringToV8(isolate, "clipboard-read");
```

Comment:
shouldn't this be `clipboard-read-write` ?

Reply:
Oh https://source.chromium.org/chromium/chromium/src/+/master:content/public/browser/permission_type.h;l=36-37 , weird permission rename in upstream.